targets/TARGET_STM/pinmap.c

```diff
@@ -79,13 +80,21 @@ void pin_function(PinName pin, int data)
 #if defined (TARGET_STM32F1)
     if (mode == STM_PIN_OUTPUT) {
 #endif
-
+		switch (speed)
+		{
```

Comment:
can you please fix formatting, follow the one in the file

```
switch (speed):
    case VALUE:
```

for 87-91 LL_ lines: they should move to be aligned with case (4 spaces to the right).

Reply:
done

Reply:
line 84 , move `{` to the line above (=attached).

line 87 : `#if` - preprocessor always at the beginning of the line (not aligned).

Reply:
done